en/concept/flight_modes.md

```diff
@@ -1,34 +1,36 @@
 # Flight Modes
 
-**Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or the [ground control station](../qgc/README.md).
+**Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or the [ground control station](../qgc/README.md), and are loosely grouped according to their level of control of the vehicle by the system vs. the user: "manual" modes, where the controller is essentially a pass-through from the user's remote control to the vehicle's actuators; "assisted" modes, where the system's outputs are coordinated with the user's commands; and, "auto" modes, where the controller requires little to no user input whatsoever.
+
+Not all flight modes are available on all vehicle types, and some modes behave differently according to vehicle type as described below. Finally, some flight modes make sense only under specific pre-flight and in-flight conditions (GPS lock, airspeed sensor, and/or vehicle attitude sensing along an axis); the system will not allow transitions to those modes until such conditions are met.
 
 ## Flight Mode Quick Summary
 
-  * **_MANUAL_**
+  * Manual flight modes
     * **Fixed wing aircraft/ rovers / boats:** 
-        * **MANUAL:** The pilot's control inputs are passed directly to the output mixer.
-        * **STABILIZED:** The pilot's inputs are passed as roll and pitch *angle* commands and a manual yaw command.
+        * **MANUAL:** (flight mode) The pilot's control inputs are passed directly to the output mixer.
```

Comment:
I personally don't think (flight mode) is needed after every mode - especially if you make the groupings into headings.